src/test/java/pikachu/task/EventTest.java

```diff
@@ -0,0 +1,17 @@
+package pikachu.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    @Test
+    public void getNameTest(){
```

Comment:
Java code normally use camelCase for method names e.g., testStringConversion but when writing test methods, sometimes another convention is used: whatIsBeingTested_descriptionOfTestInputs_expectedOutcome e.g., intDivision_zeroDivisor_exceptionThrown

Reply:
fixed after looking through the java coding standard, thank you for pointing out! I did not know about this :(